stout/event-loop.h

```diff
@@ -256,6 +293,10 @@ class EventLoop : public Scheduler {
  private:
   void Prepare();
 
+  static void CloseCallback(uv_handle_t* handle) {
+    delete handle;
+  }
```

Comment:
Does it make more sense to put this callback inside `Signal`?

Reply:
Definitely not. There are many more handles (that will be implemented in the near future) which use the same callback.